android/playground/src/main/AndroidManifest.xml

```diff
@@ -12,7 +13,8 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme" 
+        android:usesCleartextTraffic="true" >
```

Comment:
Since that's just the playground, this would never be the case in production, right?

Reply:
Right, it is only `rnkiwimobile` which is exposed to native application.